pageserver/src/http/openapi_spec.yml

```diff
@@ -273,6 +273,61 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/tenant/{tenant_id}/timeline/{timeline_id}/get_time_range_of_lsn:
```

Comment:
make sure to update https://github.com/neondatabase/cloud/blob/9b7741a2fa009b36042420dc43eeeec4432d0ea4/swagger/pageserver.yml#L275 as well